backend/database_handler/models.py

```diff
@@ -110,3 +110,22 @@ class Validators(Base):
     created_at: Mapped[Optional[datetime.datetime]] = mapped_column(
         DateTime(True), server_default=func.current_timestamp(), init=False
     )
+
+
+class LLMProviderDBModel(Base):
```

Comment:
One question that I'm thinking now is that if this should have any relationship with the validators, or if this is a pool to draw configurations with that could live independently?

Reply:
The second one